views/css/feedback.css

```diff
@@ -0,0 +1,82 @@
+/**  
```

Comment:
I have the impression most of this CSS is duplicated from other files. Do we need to duplicate most of it? Couldn't not we reuse part of it with multiple CSS imports?